documentation/modules/proc-iot-creating-project.adoc

```diff
@@ -7,7 +12,7 @@ ifeval::["{cmdcli}" == "oc"]
 . Change to a non-admin user:
```

Comment:
@dejanb Seems like the command is missing here? Or maybe the colon should just be a period here (if it depends what the exact command will be)

Reply:
@jenmalloy You're right ... it should be the same as Step 1 in https://enmasse.io/documentation/master/openshift/#create-address-space-cli-messaging-gs (I can't link to it in github properly).